riocli/deployment/list.py

```diff
@@ -78,23 +76,14 @@ def display_deployment_list(
 ):
     headers = []
     if show_header:
-        headers = ('Name', 'Status', 'Phase', 'Errors')
-        if wide:
-            headers += ('Package', 'Deployment ID',)
+        headers = ('Deployment ID', 'Name', 'Phase', 'Package', 'Creation Time (UTC)', 'Stopped Time (UTC)')
```

Comment:
todo: Add the code to display deployment errors in the list view. This was lost during the rebase.